packages/slate-tools/package.json

```diff
@@ -68,6 +68,7 @@
     "raw-loader": "^0.5.1",
     "react-dev-utils": "0.5.2",
     "sass-loader": "6.0.7",
+    "slate-locales-plugin": "file:../slate-locales-plugin",
```

Comment:
Set to `1.0.0` and run `yarn bootstrap` from the root directory to link local packages.